plugin/pxr/maya/lib/usdMaya/importTranslator.h

```diff
@@ -48,6 +48,7 @@ class UsdMayaImportTranslator : public MPxFileTranslator
 
     bool haveReadMethod() const override { return true; }
     bool haveWriteMethod() const override { return false; }
+    bool haveNamespaceSupport() const override { return true; }
```

Comment:
The ADSK import file translator is a copy-paste of the original Pixar import translator, so decided to do the fix there as well, even though no one has complained about it.  I think it is quite arguably a latent bug.  I decided not to refactor the two files and factor out the class to a common one, as the Pixar plugin should become deprecated soon.